src/app/api/v1/transfer/[id]/route.ts

```diff
@@ -0,0 +1,50 @@
+'use server';
+
+import { NextRequest, NextResponse } from 'next/server';
+import { validate } from 'uuid';
+import { SplTransfer, TransactionStatus } from '@/app/types/splTransfer';
+import { getSplTransfer } from '@/logic/transactionEngine';
+
+// This type is used to return the transaction to the client.
+// It is used to hide certain fields from the public.
+export type PublicSplTransfer = Omit<SplTransfer, 'signedTransactionBytes' | 'requestedByIp'> & {
```

Comment:
Why hide `signedTransactionBytes`?
This is going to be public on the ledger.

Reply:
honestly thinking about just removing it.. not sure what that gives us